pkg/utils/nodes.go

```diff
@@ -8,6 +8,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const WorkerLabelName = "node-role.kubernetes.io/worker"
```

Comment:
Nit: might want to add this to common repo, IIRC it is also used in SNR & NHC

Reply:
I'm wondering if this is somewhere in client-go / k8s api server code...

Reply:
I didn't find it :)

Reply:
I already have an open PR in common since it's a very small change I thought it would be ok to added that label there as well: https://github.com/medik8s/common/pull/3